integration/securitypolicy_test.go

```diff
@@ -22,36 +23,114 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
+// minPolicyAPIVersion represents the minimum Tyk Gateway version required to use Policy API.
+var minPolicyAPIVersion = version.MustParseGeneric("v4.1.0")
```

Comment:
Should we rename it to `minPolicyGwVersion`

Reply:
updated 👍